### PR TITLE
[Label Redesign] Issue#1090: Update age_load

### PR DESCRIPTION
**Pull request by Zainab-Saad**
- Add function to generate id for vertex and edge from default vertex/edge sequence
- Add label_id col for vertices and egdes created from csv
- Modify edge load function to load the respective vertex table in memory and store the table content in hashtable for vertex id 
  retrieval based on the property id
- Modified age_load regression tests and the test passes.
- Peformance is almost same as before

### CODE REVIEW

File: src/backend/utils/adt/graphid.c [hunk 1]
```diff
@@ -277,3 +280,36 @@ Datum graphid_hash_cmp(PG_FUNCTION_ARGS)
 
     PG_RETURN_INT32(hash);
 }
+
+int64 get_new_id(char entity_type, Oid graphoid) {
+    Oid seq_id;
+    int count;
+
+    // get the OID of sequence
+    if (entity_type == LABEL_KIND_VERTEX) {
```
Comment (rafsun42) [inline src/backend/utils/adt/graphid.c:289]: We keep curly brackets in a separate line. https://age.apache.org/contribution/guide/

Comment (rafsun42) [reply src/backend/utils/adt/graphid.c:289]: Also, there are extra spaces at the end of lines that needs to be removed.

**Comment by rafsun42** [comment]
@Zainab-Saad In your PR, there should be only commit related to your work on age_load. It should not include commits from the old label_redesign branch. You might find the cherry-pick command helpful.

**Comment by Zainab-Saad** [comment]
@rafsun42 I think force-pushing `label_redesign` into this branch has resulted in all these commits here. 
I am not sure how `cherry-pick` would help here. Could you plz elaborate more?
Thanks

**Comment by rafsun42** [comment]
@Zainab-Saad I know. Rebase command changes the commit hash. You can think of label_redesign as a completely different branch. So, you need to find a way to NOT include the commits in your PR that are already in rebased label_redesign. Cherry pick was just a suggestion.

**Pull request closed without merging**